test/README.md

```diff
@@ -126,3 +130,243 @@ and `setInterval`).
+
+### opensslCli
+* return [&lt;Boolean>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type)
+
+Checks whether 'opensslCli' is supported.
+
+### platformTimeout(ms)
+* `ms` [&lt;Number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type)
+* return [&lt;Number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type)
+
+Platform normalizes timeout.
+
+### PIPE
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Path to the test sock.
+
+### PORT
+* return [&lt;Number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) default = `12346`
+
+Port tests are running on.
+
+### refreshTmpDir
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Deletes the 'tmp' dir and recreates it
+
+### rootDir
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Path to the 'root' directory. either `/` or `c:\\` (windows)
+
+### skip(msg)
+* `msg` [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Logs '1..0 # Skipped: ' + `msg`
+
+### spawnCat(options)
+* `options` [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+* return [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+
+Platform normalizes the `cat` command.
+
+### spawnPwd(options)
+* `options` [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+* return [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+
+Platform normalizes the `pwd` command.
+
+### spawnSyncCat(options)
+* `options` [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+* return [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+
+Synchronous version of `spawnCat`.
+
+### spawnSyncPwd(options)
+* `options` [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+* return [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+
+Synchronous version of `spawnPwd`.
+
+### testDir
+
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Path to the 'test' directory.
+
+### tmpDir
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Path to the 'tmp' directory.
+
+### tmpDirName
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Name of the tmpDir, currently 'tmp'
```

Comment:
It's not always `tmp`.  It is initialized to that, but later on in `common.js`, it gets altered if the `TEST_THREAD_ID` environment variable is defined. I would just say: "Name of the temp directory used by tests."

Nit: period at the end for consistency